pockemun/src/api/getPokemonByTypesName.ts

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios'
+
+import type { PokemonsByType } from '@/api/types'
+
+const getPokemonsByTypeName = async (name: string) => {
```

Comment:
1. There is really no reason to create separate file for this function and `getPokemonByTypesId`, they can sit in the same file no problem
2. `baseUrl` should be exported as global constant